lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -150,24 +171,6 @@ static void addMandatoryDiagnosticOptPipeline(SILPassPipelinePlan &P) {
   // End Ownership Optimizations
   //===---
 
-  P.addAddressLowering();
-
-  P.addFlowIsolation();
-
-  // Automatic differentiation: canonicalize all differentiability witnesses
-  // and `differentiable_function` instructions.
-  P.addDifferentiation();
-
-  // Only run semantic arc opts if we are optimizing and if mandatory semantic
-  // arc opts is explicitly enabled.
-  //
-  // NOTE: Eventually this pass will be split into a mandatory/more aggressive
-  // pass. This will happen when OSSA is no longer eliminated before the
-  // optimizer pipeline is run implying we can put a pass that requires OSSA
-  // there.
-  const auto &Options = P.getOptions();
```

Comment:
Can you add a test that shows this behavior? Then I am happy.

Reply:
I talked with JoeG offline about this. They mentioned that this is just the first part of a larger fix so it isn't time yet to add a test for this behavior. So... LGTM!

Reply:
There is no behavior change yet.